### PR TITLE
rendered countries and their details

### PR DESCRIPTION
**Pull request by Developer-Kingz**
- created a home component to render list of countries from API
- added a details component to view more details about a country
- added redux slice for state management
- added linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,159 @@
...
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+-   **[make the calculator functional]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- Demo Video: [Demo Video] (https://www.loom.com/share/955c0e9e766d4a9eadc9094c6c9d5546)
```
Comment (mwanawabangona) [inline README.md]: - [ ] `Required`
- [ ] Amazing work 👏 
- [ ] The maximum length of the video is 5 minutes. ✔️ 
- [ ] The video contains a clear and objective description of the project in professional English ✔️ 
- [ ] The video contains a project demo, highlighting all the project features. ✔️ 
- [ ] The video highlights a piece of the code the `student` finds interesting or is proud of making. ❌ 
Please make the video contain highlights of a piece of code you find interesting or proud of making
According to the requirements. 🥂

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  }
+
+  .details-cont-inner > :nth-child(even) {
+    background-color: rgb(32, 57, 57);
+  }
+
+  .details-cont-inner > :nth-child(odd) {
+    background-color: rgb(89, 143, 143);
+  }
+}
```
Comment (mwanawabangona) [inline src/App.css]: - [ ] `Required`
- [ ] Colors (select one main color for your website). ✔️ 
- [ ] Typography: font face, size and weight. ✔️ 
- [ ] However the  Layout: composition missed a few elements. 😞 
- [ ] kindly add the missing elements
- [ ] please sceenshoots below and the highlighted areas 👇 

`original`
image

`your design`

image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Developer-Kingz ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @Developer-Kingz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] All requested changes are implemented ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**